.github/workflows/feature-branch-prerelease.yml

```diff
@@ -0,0 +1,114 @@
+name: Prerelease feature branch
+
+on:
+  workflow_dispatch:
+    inputs:
+      featureBranch:
+        type: string
+        description: Feature branch name
+        required: true
```

Comment:
Can you please get the branch name based on the current branch name to prevent typos?

<img width="345" alt="image" src="https://github.com/amplitude/Amplitude-TypeScript/assets/5790872/f5367af9-947d-4d35-b016-46098c57a0f9">


I found a 3rd party action you can use:
https://github.com/tj-actions/branch-names

Reply:
Use the third party action and delete branch name input at https://github.com/amplitude/Amplitude-TypeScript/pull/621/commits/6e6ae31594326a131173b7148828b7765a1201f5